packages/shared/components/ProposalQuestion.svelte

```diff
@@ -24,8 +24,7 @@
     $: answers = [...question?.answers, { value: 0, text: 'Abstain', additionalInfo: '' }]
     $: percentages = getPercentagesFromAnswerStatuses(answerStatuses)
     $: disabled =
-        $selectedProposal.state?.status === ProposalStatus.Upcoming ||
-        $selectedProposal.state?.status === ProposalStatus.Ended
+        $selectedProposal?.status === ProposalStatus.Upcoming || $selectedProposal?.status === ProposalStatus.Ended
```

Comment:
do we need the `?` after the `$selectedProposal` because we're only able to come to this page if we have a `selectedProposal`

Reply:
probably not

Reply:
But I added this everywhere and there is no harm right